### PR TITLE
Request cards

### LINKED ISSUE #70: Change Meta Title fields
- [ ] Change Meta title on each page visited

### LINKED ISSUE #76: Add UI to request pages

### PR DESCRIPTION
**Pull request by edmdc**
# Pull request checklist

## Pull request type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## Current behavior

- Request cards are messy
- functionality is unnecessarily spread out where
- Campaigns is a weighted word that the team doesn't like.

## New behavior

- Functionality has been condensed into two request cards (manager & contributor) from three.
- Request card styles have been standardized.
- Campaign has been replaced to Booster on the main nav header. New Booster/Campaign link has been moved to user boosters

## Other information

**New Nav Header**
image

**New Request Card**
image

Closes #70 
Closes #76

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -8,9 +8,8 @@
         </router-link>
       </div>
       <div class="links-section">
-        <router-link to="/">Campaigns</router-link>
-        <router-link to="/campaigns/new"> Create New Campaign</router-link>
-        <router-link to="/campaigns/user">Your Campaigns</router-link>
+        <router-link to="/">Boosters</router-link>
```
Comment (jpc20) [inline src/App.vue:11]: Nice job switching to Booster already.

**Review by jpc20 — APPROVED** [review]
Great job refactoring this Edwin. I just ran everything locally and it seems to be running smoothly.

File: src/components/MyCreatedCampaigns.vue [hunk 2]
```diff
@@ -1,38 +1,27 @@
...
-        <router-link
-          style="text-decoration: none;"
-          :to="{name: 'Edit Campaign',
-            params: { address: campaign.address } }">Edit
-        </router-link>
-        <a href="javascript:;" v-on:click="deleteCampaign(campaign)">Delete</a>
-      </section>
-    </div>
+    <ul class="user-campaigns" v-for="campaign in userCampaigns" v-bind:key="campaign.id">
+      <request-list
```
Comment (attom2) [inline src/components/MyCreatedCampaigns.vue:3]: This is some dope modularity!

**Review by attom2 — APPROVED** [review]
Looks great! I like the reusability of components

**Pull request merged**